### PR TITLE
Authenticating Torus Users Section

### PR DESCRIPTION
**Pull request by 1swaraj**

### CODE REVIEW

File: docs/authenticating-torus-users/torus-users.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  "verifier_id": "hello@tor.us",
+  "oauth_pub_key": "0xBEEF4321...",
+  "app_scoped_pubkey": "0x1234C0DE...",
+  "network": "mainnet"
+}
+```
+
+##### Challenge Generation
+
+1. Store the following data in a database / cache
```
Comment (himanshuchawla009) [inline docs/authenticating-torus-users/torus-users.mdx:26]: why we need to write internal working of challenge generation part here? This should be internal right? We should simply write endpoint of getting challenge with req and res params explanation.

File: docs/authenticating-torus-users/torus-users.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  "signature": "304402203a5b354ebc147..." // Hex encoded signature
+}
+```
+
+> Code: `200 OK`
+
+```json
+{
+  "success": true,
+  "token": "eyJhbGciOiJFUz..." // JWT token
```
[the remark this answers is not in the document]
Comment (1swaraj) [reply docs/authenticating-torus-users/torus-users.mdx:65]: Added a separate page for this

File: docs/authenticating-torus-users/torus-users.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  "challenge": "message-to-be-signed-0000"
+}
+```
+
+#### Verify signed challenge
+
+1. Get the challenge from the database using the key : `${oauth_pub_key}_CHALLENGE`. If the verify function is called after 60 seconds, the challenge
+   doesn't exist as it is removed from the database due to the timeout set.
+
+2. Verify the signature against the challenge obtained in step 1 using the `oauth_pub_key`
```
[the remark this answers is not in the document]
Comment (1swaraj) [reply docs/authenticating-torus-users/torus-users.mdx]: Added

File: docs/authenticating-torus-users/torus-users.mdx (hunk 1 above)
Comment (1swaraj) [reply docs/authenticating-torus-users/torus-users.mdx]: Makes sense.. It can be abstracted from the user..

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Pull request merged**